### ISSUE TITLE
Node Root Volume Size Overrides [gradient-aws-kubernetes]

### ISSUE THREAD

**Issue by marquiswashere** [Paperspace/gradient-installer#243]
```
    node_volume_sizes = merge({
        "services-small"=local.root_volume_size_default,

        "experiment-cpu-small"=local.root_volume_size_default,
        "experiment-cpu-medium"=local.root_volume_size_default,

        "model-deployment-cpu-small"=local.root_volume_size_default,
        "model-deployment-cpu-medium"=local.root_volume_size_default,

        "notebook-cpu-small"=local.root_volume_size_default,
        "notebook-cpu-medium"=local.root_volume_size_default,

        "tensorboard-cpu-small"=local.root_volume_size_default,
        "tensorboard-cpu-medium"=local.root_volume_size_default,
    }, var.node_asg_max_sizes)
  ```
  
Currently we're allowing `var.node_asg_max_sizes` to be able to potentially override our EC2 instances' root volume size.  I think this is a bug.

**Comment by paperspacemarquis** [comment]
Further inspection we don't actually pipe this variable's contents. This will be closed by an upcoming PR.

**Comment by marquiswashere** [comment]
Closed by #224
